test/components/structures/auth/Registration-test.js

```diff
@@ -1,105 +0,0 @@
-/*
```

Comment:
it's a little concerning that our only registration tests are RTS ones...

Reply:
Yes, I am not exactly thrilled about it, especially since I am actively changing it and it's a critical path of the app... 😰 

Wish I had more time to add lots of tests...